charts/rancher-logging/0.1.2/charts/log-aggregator/charts/log-aggregator-linux/templates/daemonset.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: {{ template "daemonset_api_version" . }}
 kind: DaemonSet
 metadata:
-  name: {{ template "log-aggregator.fullname" . }}
+  name: {{ template "log-aggregator.fullname" . }}-linux
```

Comment:
I don't think you can change the name of a daemonset on upgrade.

Reply:
Because we separate to Linux and Windows, so we add a suffix for that Daemonset, we use force upgrade which will remove the old chart and install the new chart. Because some changes between two version charts may cause upgrade failed, so we use force upgrade, for example, we remove label chart under `spec.selector.matchLabels`, and matchLabels is an unmodifiable field in `apps/v1` Daemonset.

Reply:
Have you explicitly tested upgrade scenarios around this and verified it works as expected for linux? I need confirmation on that before merging.

Reply:
Yes, I have tested the upgrade scenarios.